services/shared/src/types/zod.ts

```diff
@@ -138,6 +143,29 @@ export const zodExtendedCommentData = z
         commentItem: zodCommentItem,
     })
     .strict();
+export const zodReportReason = z.union([
```

Comment:
For the updated list of reasons, check out the Figma:
https://www.figma.com/design/AD1cYjYTAz47YRInLqrLAV/Agora-Citizen-Network-Prototype?node-id=17538-58924&t=YnaGwKpPmaxIrjKG-0